hr_employee_firstname/models/hr.py

```diff
@@ -0,0 +1,81 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2010 - 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class hr_employee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.cr_context
+    def _auto_init(self, cr, context=None):
+        res = super(hr_employee, self)._auto_init(cr, context=context)
+        cr.execute("""
```

Comment:
I will add a test...the ideea is that if you already have employee's in database, and install after the module, it will split the name_related fields to firstname and lastname fields for the ones that are recorded...i've put the calculation about "-2" is to check if by mistake the employee's name doesn't start with a " ", and if it start's to pass it.